src/mock-json/example4.json

```diff
@@ -0,0 +1,87 @@
+    ["prefix_2: \"administrative fee - the first pol\"", 0.1],
+    ["Activity code_2: \"AC419100\"", 0.1],
+    ["Number of executions_2: ", 0.1],
+    ["Producer code_2: \"SRTH\"", 0.1],
+    ["Section_2: \"Section 5\"", 0.1],
+    ["Specialism code_2: \"SC61\"", 0.1],
+    ["group_2: \"Radiotherapy\"", 0.1],
+    ["lifecycle:transition_2: \"complete\"", 0.1],
+    ["prefix_3: \"verlosk.-gynaec. short-out card cost\"", 0.1],
+    ["Activity code_3: \"AC10107\"", 0.1],
+    ["Number of executions_3: ", 0.1],
+    ["Producer code_3: \"SGEH\"", 0.1],
+    ["Section_3: \"Section 2\"", 0.1],
+    ["Specialism code_3: \"SC7\"", 0.1],
+    ["group_3: \"Nursing ward\"", 0.1],
+    ["lifecycle:transition_3: \"complete\"", 0.1],
+    ["prefix_4: \"ultrasound - internal genitals\"", 0.1],
+    ["Activity code_4: \"339486E\"", 0.1],
+    ["Number of executions_4: ", 0.1],
+    ["Producer code_4: \"SGEC\"", 0.1],
+    ["Section_4: \"Section 2\"", 0.1],
+    ["Specialism code_4: \"SC7\"", 0.1],
+    ["group_4: \"Obstetrics & Gynaecology clinic\"", 0.1],
+    ["lifecycle:transition_4: \"complete\"", 0.1],
+    ["prefix_5: \"First outpatient consultation\"", 0.1],
+    ["Activity code_5: \"AC410100\"", 0.1],
+    ["Number of executions_5: ", 0.1],
+    ["Producer code_5: \"SGEH\"", 0.1],
+    ["Section_5: \"Section 2\"", 0.1],
+    ["Specialism code_5: \"SC7\"", 0.1],
+    ["group_5: \"Nursing ward\"", 0.1],
+    ["lifecycle:transition_5: \"complete\"", 0.1],
+    ["prefix_6: \"administrative fee - the first pol\"", 0.1],
+    ["Activity code_6: \"AC419100\"", 0.1],
+    ["Number of executions_6: ", 0.1],
+    ["Producer code_6: \"SGEH\"", 0.1],
+    ["Section_6: \"Section 2\"", 0.1],
+    ["Specialism code_6: \"SC7\"", 0.1],
+    ["group_6: \"Nursing ward\"", 0.1],
+    ["lifecycle:transition_6: \"complete\"", 0.1],
+    ["prefix_7: \"simulator - use before megavol\"", 0.1],
+    ["Activity code_7: \"AC390520\"", 0.1],
+    ["Number of executions_7: ", 0.1],
+    ["Producer code_7: \"RATH\"", 0.1],
+    ["Section_7: \"Section 5\"", 0.1],
+    ["Specialism code_7: \"SC61\"", 0.1],
+    ["group_7: \"Radiotherapy\"", 0.1],
+    ["lifecycle:transition_7: \"complete\"", 0.1],
+    ["prefix_8: \"treatment time - Unit t3 - megavolt\"", 0.1],
+    ["Activity code_8: \"AC390003\"", 0.1],
+    ["Number of executions_8: ", 0.1],
+    ["Producer code_8: \"RATH\"", 0.1],
+    ["Section_8: \"Section 5\"", 0.1],
+    ["Specialism code_8: \"SC61\"", 0.1],
+    ["group_8: \"Radiotherapy\"", 0.1],
+    ["lifecycle:transition_8: \"complete\"", 0.1],
+    ["prefix_9: \"teletherapy - megavolt photons bestrali\"", 0.1],
+    ["Activity code_9: \"AC390550\"", 0.1],
+    ["Number of executions_9: ", 0.1],
+    ["Producer code_9: \"RATH\"", 0.1],
+    ["Section_9: \"Section 5\"", 0.1],
+    ["Specialism code_9: \"SC61\"", 0.1],
+    ["group_9: \"Radiotherapy\"", 0.1],
+    ["lifecycle:transition_9: \"complete\"", 0.1]
+    ]
+  }
+]
+}
```

Comment:
the src folder it is supposed to contain only code, move this outside the src folder